src/Extensions/FormEmailEncryptionExtension.php

```diff
@@ -58,7 +58,7 @@ public function updateCMSFields(FieldList $fields): FieldList
      *
      * @return bool
      */
-    public function encryptEmail(): bool
+    public function isEncryptEmail(): bool
```

Comment:
Updating to more logical naming and avoid naming conflict with other encryption functions